examples/telescope/components/react/vote.tsx

```diff
@@ -0,0 +1,224 @@
+import React, { useCallback, useEffect, useState } from 'react';
+import { ChainName } from '@cosmos-kit/core';
+import {
+  Box,
+  Center,
+  Heading,
+  Spinner,
+  Stack,
+  useDisclosure,
+  Text,
+  useColorModeValue,
+} from '@chakra-ui/react';
+import { ProposalCard } from './proposal-card';
+import { ProposalModal } from './proposal-modal';
+import { useChain } from '@cosmos-kit/react';
+import { cosmos } from '../../codegen';
+import * as gov from '../../bufcodegen/cosmos/gov/v1beta1/gov_pb';
+import * as query from '../../bufcodegen/cosmos/gov/v1beta1/query_pb';
```

Comment:
this is using `bufcodegen` too

Reply:
I believe it's been removed till the last commit too.